src/html/classic/ng/src/web/pages/extras/trashcanpage.js

```diff
@@ -0,0 +1,493 @@
+        <AgentsTable
+          entities={agent_list}
+          {...table_props}
+        />
+
+        <LinkTarget id="Alerts"/>
+        <h1>{_('Alerts')}</h1>
+        <AlertsTable
+          entities={alert_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Configs"/>
+        <h1>{_('Scan Configs')}</h1>
+        <ScanConfigsTable
+          entities={scan_config_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Credentials"/>
+        <h1>{_('Credentials')}</h1>
+        <CredentialsTable
+          entities={credential_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Filters"/>
+        <h1>{_('Filters')}</h1>
+        <FiltersTable
+          entities={filter_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Groups"/>
+        <h1>{_('Groups')}</h1>
+        <GroupsTable
+          entities={group_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Notes"/>
+        <h1>{_('Notes')}</h1>
+        <NotesTable
+          entities={note_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Overrides"/>
+        <h1>{_('Overrides')}</h1>
+        <OverridesTable
+          entities={override_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Permissions"/>
+        <h1>{_('Permissions')}</h1>
+        <PermissionsTable
+          entities={permission_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Port Lists"/>
+        <h1>{_('Port Lists')}</h1>
+        <PortListsTable
+          entities={port_list_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Report Formats"/>
+        <h1>{_('Report Formats')}</h1>
+        <ReportFormatsTable
+          entities={report_format_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Roles"/>
+        <h1>{_('Roles')}</h1>
+        <RolesTable
+          entities={role_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Scanners"/>
+        <h1>{_('Scanners')}</h1>
+        <ScannersTable
+          entities={scanner_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Schedules"/>
+        <h1>{_('Schedules')}</h1>
+        <SchedulesTable
+          entities={schedule_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Tags"/>
+        <h1>{_('Tags')}</h1>
+        <TagsTable
+          entities={tag_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Targets"/>
+        <h1>{_('Targets')}</h1>
+        <TargetsTable
+          entities={target_list}
+          {...table_props}
+         />
+
+        <LinkTarget id="Tasks"/>
+        <h1>{_('Tasks')}</h1>
+        <TasksTable
+          entities={task_list}
+          {...table_props}
+         />
+
+      </Layout>
+    );
+  }
+};
+
+Trashcan.propTypes = {
+  gmp: PropTypes.gmp.isRequired,
+};
+
+export default withContext({
```

Comment:
you can replace this with withGmp HOC now. I have added a withGmp HOC some weeks ago.
```javascript
export default withGmp(Tashcan);
```

Reply:
Done